src/docfx/SubCommands/BuildCommand.cs

```diff
@@ -338,33 +367,39 @@ private static void MergeGitContributeToConfig(BuildJsonConfig config)
                 }
             }
 
-            if (globalMetadataFilePath != null)
+            if (globalMetadataFilePaths != null)
             {
-                try
+                foreach (var globalMetadataFilePath in globalMetadataFilePaths)
                 {
-                    var globalMetadataFromFile = JsonUtility.Deserialize<BuildJsonConfig>(globalMetadataFilePath).GlobalMetadata;
-                    if (globalMetadataFromFile == null)
+                    Dictionary<string, object> metadata = null;
+                    try
+                    {
+                        metadata = JsonUtility.Deserialize<BuildJsonConfig>(globalMetadataFilePath).GlobalMetadata;
```

Comment:
>BuildJsonConfig [](start = 59, length = 15)

this means global metadata file can contain file metadata section but will be ignored?